cirq/testing/circuit_compare.py

```diff
@@ -148,3 +163,37 @@ def assert_circuits_with_terminal_measurements_are_equivalent(
         'Diagram of reference circuit with desired function:\n'
         '{}\n'.format(actual, reference)
     )
+
+
+def assert_same_diagram(
+        actual: circuits.Circuit,
+        desired: str,
+        use_unicode_characters: bool = True,
```

Comment:
Add a `*` before `use_unicode_characters` to force keyword arguments on the options.

You may want to take a `**kwargs` argument and forward it into `to_text_diagram`, instead of listing specific ones, to future-proof against features being added to the text diagram methods.